### ISSUE TITLE
demo website https://abboom.com asking username and password

### ISSUE THREAD

**Issue by kamleshwebtech** [kunjara/jyotish#17]
When we access demo website https://abboom.com to see this repository demo. It asks username and password to enter and display demo website. Please share username and password Or remove this restriction so that we can see demo of this application. Thanks a lot. Look forward to hear you.

**Comment by jscuba** [comment]
Looks like abboom.com was affected when @kunjara stopped maintaining the repo. [Abboom.com founder comment](https://github.com/kunjara/jyotish/issues/7#issuecomment-405091916)

If you want a demo site, you can try [sidereal.live](https://sidereal.live) It uses this library.

Kuddos to @kunjara for building a library that still works 3 years later without any updates.

**Comment by mahavira9** [comment]
hello @jscuba thank you for your brilliant comment, actually it’s been 3 years i'm struggling with this
do you know website owner? i want to buy full & currect script, i'm tired , i'm astrologist ,i dont have programming knowledge , from iran, regards

**Comment by Ardalin** [comment]
> hello @jscuba thank you for your brilliant comment, actually it’s been 3 years i'm struggling with this
> do you know website owner? i want to buy full & currect script, i'm tired , i'm astrologist ,i dont have programming knowledge , from iran, regards

What exactly fo you need?
Im the founder of abboom, been in contact with kunjara about month ago and in contact with Anton in Australia who is keeping deep development/rebirth of the product.

**Comment by mahavira9** [comment]
> > hello @jscuba thank you for your brilliant comment, actually it’s been 3 years i'm struggling with this
> > do you know website owner? i want to buy full & currect script, i'm tired , i'm astrologist ,i dont have programming knowledge , from iran, regards
> 
> What exactly fo you need?
> Im the founder of abboom, been in contact with kunjara about month ago and in contact with Anton in Australia who is keeping deep development/rebirth of the product.

thank you for your help , i want to build a website like sidereal.live in persian but i'm astrologist and don't  know much j about programming , i taught maybe i could buy sidereal.live source from owner and customise it for my self, please it’s  3  been 3 years and i really need something like sidereal.live 
regards

**Comment by jscuba** [comment]
@mahavira9 that’s an interesting idea. Having the source wouldn’t give you a working website though. Maybe I can work with you to get you what you want. My server exposes the jyotish library as an API. I could share the API. We can talk about pricing. I would maintain the server and API and you could use it for a monthly fee. If you want a Persian version of sidereal.live, that’s a web development project that you could take on yourself using the API or we could internationalize sidereal.live if you want to do that. Contact me at info@sidereal.live and we can talk more about it.

**Comment by mahavira9** [comment]
> @mahavira9 that’s an interesting idea. Having the source wouldn’t give you a working website though. Maybe I can work with you to get you what you want. My server exposes the jyotish library as an API. I could share the API. We can talk about pricing. I would maintain the server and API and you could use it for a monthly fee. If you want a Persian version of sidereal.live, that’s a web development project that you could take on yourself using the API or we could internationalize sidereal.live if you want to do that. Contact me at [info@sidereal.live](mailto:info@sidereal.live) and we can talk more about it.

i emailed you 4 days ago and still no answers

**Comment by jscuba** [comment]
Yes, you want buy source code, but the source code for the server is already available on this repo. The rest of the setup is server configuration with the library that’s here, and then building a website that uses it. You mentioned that you have a developer working with you. Perhaps the developer can set that up for you. There are many ways to setup the server with this library.

**Comment by mahavira9** [comment]
> Yes, you want buy source code, but the source code for the server is already available on this repo. The rest of the setup is server configuration with the library that’s here, and then building a website that uses it. You mentioned that you have a developer working with you. Perhaps the developer can set that up for you. There are many ways to setup the server with this library.

in iran we have some problem , can you do this for me? , or can i buy API ready from you? please help

**Comment by jscuba** [comment]
That's not something I want to work on.
